lib/Onyx.js

```diff
@@ -1106,11 +1106,14 @@ function clear(keysToPreserve = []) {
                 notifyCollectionSubscribersOnNextTick(key, value);
             });
 
-            // Call clear() and make sure that the default key/values and the key/values from the parameter
-            // are preserved in storage. This makes sure to always leave storage in a state that contains
-            // all the default values and any additional values that we want to remain after the database is cleared.
```

Comment:
Let's keep a comment here to explain what we're doing and why

Reply:
@madmax330 Thanks for pointing that out. I have updated, please help to check again

Reply:
updated it to only say _what_ and not why...